src/web/rustdoc.rs

```diff
@@ -604,12 +602,6 @@ pub(crate) async fn rustdoc_html_server_handler(
         .recently_accessed_releases
         .record(krate.crate_id, krate.release_id, target);
 
-    let target = if target.is_empty() {
-        String::new()
-    } else {
-        format!("{target}/")
```

Comment:
I'm surprised as well. Do we have tests to ensure this behaviour by any chance? Because `target` is declared in the templates but I'd like to be sure it's actually doing what it's supposed to.

Reply:
this is super odd. From what I see the only place where this attribute is used is in the releases-list, where it gets contatenated with the `inner_path` without a slash in between. So we would nees this trailing slash here when the inner path doesn't start with a slash. 

But I fail to find any situation where it's either broken on master or in this PR, and I don't know why. 

So my hunch is to leave it out and see if we find breakages.

Reply:
Ah wait, you're not even passing it into the `RustdocPage` any more. 

So, the reason we don't need it is that the releases-list is not directly rendered any more, but only via ajax, in `web::crate_details::get_all_releases`, 

where we still have this behaviour for the target string.